src/app/GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -113,6 +113,9 @@ public FormRemotes(IGitUICommands commands)
             // Set the minimum height of the color button to be the same as the reset button
             btnRemoteColor.MinimumSize = new Size(DpiUtil.Scale(60), btnRemoteColorReset.Height);
 
+            // Prevent btnRemoteColorReset from blinking out soon after form loads
+            btnRemoteColorReset.Visible = false;
```

Comment:
Set in the designer file please.

Reply:
The reason it wasn't set in a designer is that a few lines above its height is used to set `MinimumSize` of `btnRemoteColor` and if it's invisible during component initialization it has a wrong height.

I've changed the `MinimumSize` calculation of `btnRemoteColor` to use its own height as this is before index change event happens and this button's props are changed.